### PR TITLE
Creates sum_equals_xor.py

### PR DESCRIPTION
**Pull request by ManasGupta1998**

### CODE REVIEW

File: code/bit-manipulation/sum_equals_xor/sum_equals_xor.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Part of Cosmos by OpenGenus Foundation 
+
+# Counts the number of values between 0 and a given number that satisfy the condition x+n = x^n (0<=x<=n)
+
+# Works for both python 2 and python 3
+
+n=int(input())
```
Comment (arnavb) [inline code/bit-manipulation/sum_equals_xor/sum_equals_xor.py]: Add a prompt, eg "Enter a number: ".

File: code/bit-manipulation/sum_equals_xor/sum_equals_xor.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+print (int(pow(2,(bin(n)[2:]).count('0'))))
```
Comment (arnavb) [inline code/bit-manipulation/sum_equals_xor/sum_equals_xor.py]: An explanatory comment would be helpful.

File: code/bit-manipulation/sum_equals_xor/sum_equals_xor.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Part of Cosmos by OpenGenus Foundation
```
Comment (arnavb) [inline code/bit-manipulation/sum_equals_xor/sum_equals_xor.py]: This comment is no longer necessary.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**